.github/workflows/tests.yml

```diff
@@ -0,0 +1,29 @@
+# Authored by Alexandra N. Walker <alex.walker@indicio.tech> ☀️
+
+name: tests
+"on":
+  push:
+    branches: 
+      - main
+  pull_request:
+    branches:
+      - main
+      - feature/**
+      - feat/**
```

Comment:
Nitpick: I prefer the naming scheme `feature/**` over `feat/**` (though `feat: adsfasdf` is what we use in the commit message itself); I think we should remove the `feat/**` to help enforce that convention :slightly_smiling_face: